### DISCUSSION TITLE
【功能建议】有没有大佬考虑做个学习版的b站首页

### DISCUSSION THREAD

**Discussion by Ysenxxx** [the1812/Bilibili-Evolved#4218] [功能建议 / Ideas]
经常用b站学习，但是老控制不住刷视频，就想要一个这样的首页
就只显示关注的，还有收藏夹的内容，还有搜索功能，不能推荐视频，视频界面里右边也不会推荐视频

**Comment by Geeyun-JY3** [comment]
感觉现有的几个组件已经基本覆盖了你的需求。

> 就只显示关注的

v2 的`极简首页`已经能让首页只有`动态`和`热门`两个 tab，你将默认 tab 设为`动态`就是你要的效果。
要是还控制不住自己回去点`热门` tab，那你也可以问问能不能再加这么个选项……

> 还有收藏夹的内容，还有搜索功能，不能推荐视频

用`自定义顶栏`：

- `自定义顶栏`的「布局设置」里只显示`收藏`和`搜索`；
  - 之前也有过这么一个建议：#1546
- 脚本的`自定义顶栏`本身就是没有推荐；
- 要是希望搜索结果页面的 `bilibili 热搜`也隐藏掉，预览版有个`隐藏热搜`组件

> 视频界面里右边也不会推荐视频

装`隐藏视频推荐`、`隐藏直播推荐`

**Reply by Tinhone** [reply]
感觉这可以做个合集包欸
